npm/translatePolicy.go

```diff
@@ -273,10 +278,21 @@ func translateIngress(ns string, targetSelector metav1.LabelSelector, rules []ne
 		}
 
 		// fromRuleExists
-		for _, fromRule := range rule.From {
+		for j, fromRule := range rule.From {
 			// Handle IPBlock field of NetworkPolicyPeer
 			if fromRule.IPBlock != nil {
 				if len(fromRule.IPBlock.CIDR) > 0 {
+					ipCidrs[i] = append(ipCidrs[i], fromRule.IPBlock.CIDR)
+					cidrIpsetName := name + "-cidr" + strconv.Itoa(i) + "in"
```

Comment:
no need for "-cidr", but add namespace

Reply:
OK.

Reply:
What about keeping the `cidr` to be more explicit? We just want this name to be unique.
 At least we have to worry other ipset formed by podSelector or namespaceSelector from the same networkPolicyPeer in the future.
```
type NetworkPolicyPeer struct {
	PodSelector *metav1.LabelSelector 

	NamespaceSelector *metav1.LabelSelector

	IPBlock *IPBlock 
}
```

Reply:
Yes, keep cidr will make it more explicit but it's not a hard request for now. Ipset also has a limitation for the name string length.
Current NPM keeps ipset for pod and namespace already. For pod, npm uses pod label key, pod label key and value as ipset name. For namespace, upm uses the name of namespaces as ipset name. They are not conflicting with IPBlock ipset name.